src/contexts/MessageContext/index.tsx

```diff
@@ -38,7 +46,9 @@ const MessageProvider = ({ children }: { children: ReactNode }) => {
         message,
         showMessage: setMessage,
         showMessageOnNextPage: setQueuedMessage,
-        clearMessage
+        clearMessage,
+        errorMessageInModal,
+        showErrorMessageInModal: setErrorMessageInModal
```

Comment:
I created a separate error message to be used in modals as we now are having more situations were the error message shows up in the modal.

This differentiates the messages and mainly addressing double error messages showing up when we only use `message` and `showMessage`:
![image](https://github.com/user-attachments/assets/ff2d4561-fde8-4e5e-88a8-994e8ee278dd)
 

`errorMessageInModal` in use:

![94957](https://github.com/user-attachments/assets/9132d825-e42c-4c87-b94d-66e8454da784)